app/models/ability.rb

```diff
@@ -0,0 +1,35 @@
+class Ability
```

Comment:
Looking forward to hearing about Cancan gem and its abilities.

Reply:
Basically, it just allows you to restrict who has access to what. In our case, it's restricting who has access to updating and deleting reviews - only the person who wrote them has those capabilities.